### ISSUE TITLE
Bug located in doc-build (windows)

### ISSUE THREAD

**Issue by SMoraisAnsys** [ansys/actions#622]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

When one uses the latest version of `ansys/actions/doc-build` in windows, there is an additional step that was introduced in #593. However the implementation leverage an environment variable that does not exist if one doesn't activate the upload assets input.

I'll open the floor for others things that could lead to other issues / devs:

1. Are we sure that the current workflows avoids pdf build from being performed ? AFAIK, the changes of #593 do not prevent to build the pdf (and consume CICD time), they only check before uploading. If we agree with that, I'll propose some changes in the action.
2. Couldn't this checking step be added to _doc-build-linux ?

### 📝 Steps to reproduce

See the logs of https://github.com/ansys/pyansys-sound/actions/runs/11289857973/job/31666254238

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

out of topic

### 🐍 Which Python version are you using?

3.12

### 📦 Installed packages

```shell
out of topic
```

**Comment by MaxJPRey** [comment]
Nice catch @SMoraisAnsys

**Comment by RobPasMue** [comment]
We should ignore the check if the user requested to ignore PDF build

**Comment by SMoraisAnsys** [comment]
@RobPasMue I'm not sure I understand your comment. Currently the action does not allow to ignore PDF build. Would you like to add that feature ?

**Comment by RobPasMue** [comment]
> [@RobPasMue](https://github.com/RobPasMue) I'm not sure I understand your comment. Currently the action does not allow to ignore PDF build. Would you like to add that feature ?

No, you are right... I thought that at some point we were giving the option to also skip the PDF build. But we decided to avoid this so that we made sure that all projects have a PDF version of the docs. My bad - sorry @SMoraisAnsys
